test/unit/cpu/test-scan.cxx

```diff
@@ -20,10 +20,20 @@ const int N = 1024;
 
 // Unit Test Space Exploration
 
-#ifdef RAJA_ENABLE_OPENMP
-using ExecTypes = std::tuple<RAJA::seq_exec, RAJA::omp_parallel_for_exec>;
-#else
-using ExecTypes = std::tuple<RAJA::seq_exec>;
+// We have to do all of them separately like this so the
+// Cross works; otherwise we end up with too many template arguments
+// and it kills the compiler
+//
+// Doing all of these _drastically_ increases compilation time.
```

Comment:
As mentioned in the lead comment, trying to make one tuple with all of the execution policies and then crossing it with the reduction policies breaks the build because it exceeds the max number of template parameters, at least with gcc 5.2-6.2. Unfortunately, doing it this way adds a lot to compile-time but I didn't see a better solution that didn't involve even more repetition.